src/ert/dark_storage/common.py

```diff
@@ -36,21 +38,20 @@ def data_for_key(
     if key.startswith("LOG10_"):
         key = key[6:]
     if key in ensemble.get_summary_keyset():
-        data = res.load_all_summary_data(ensemble, [key], realization_index)
+        data = ensemble.load_all_summary_data([key], realization_index)
```

Comment:
Think we should try to use the ensemble directly here, something like:
```python
ensemble.load_response(key, realization_index)
...
```

Suggest the same for GEN_KW, and then do the manipulation after we load the dataset.